### PR TITLE
feat(payment-services): Change format of payment service IDs

### PR DESCRIPTION
**Pull request by cbetta**
Updates the payment service's to use a UUID instead of a custom format for its ID, in line with other APIs.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: reference/request-bodies/card_rule_create_request.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ properties:
       A list of IDs for the payment services to use, in order of priority.
       The payment services all need to process cards.
     example:
-      - stripe-card-f47db210-8e30-4f74-8123-b834604f2042
+      - f47db210-8e30-4f74-8123-b834604f2042
       - card-braintree-0bb93ab0-86ef-4ad5-addf-b69913128d96
```
Comment (danielknell) [inline reference/request-bodies/card_rule_create_request.yaml:62]:
```suggestion
      - 0bb93ab0-86ef-4ad5-addf-b69913128d96
```

File: reference/request-bodies/card_rule_update_request.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ properties:
       A list of IDs for the payment services to use, in order of priority.
       The payment services all need to process cards.
     example:
-      - stripe-card-f47db210-8e30-4f74-8123-b834604f2042
+      - f47db210-8e30-4f74-8123-b834604f2042
       - card-braintree-0bb93ab0-86ef-4ad5-addf-b69913128d96
```
Comment (danielknell) [inline reference/request-bodies/card_rule_update_request.yaml:57]:
```suggestion
      - 0bb93ab0-86ef-4ad5-addf-b69913128d96
```

File: reference/resources/card_rule.yaml [hunk 3]
```diff
@@ -60,7 +60,7 @@ properties:
       A list of IDs for the payment services to use, in order of priority.
       The payment services all need to process cards.
     example:
-      - stripe-card-f47db210-8e30-4f74-8123-b834604f2042
+      - f47db210-8e30-4f74-8123-b834604f2042
       - card-braintree-0bb93ab0-86ef-4ad5-addf-b69913128d96
```
Comment (danielknell) [inline reference/resources/card_rule.yaml:64]:
```suggestion
      - 0bb93ab0-86ef-4ad5-addf-b69913128d96
```

**Review by danielknell — CHANGES_REQUESTED** [review]
do we need to look at these too the linting errors in reference/openapi.yaml too?

**Comment by cbetta** [comment]
@danielknell no the linting warnings are to be expected.

**Review by danielknell — APPROVED** [review]

**Pull request merged**